redpanda/values.yaml

```diff
@@ -311,6 +359,68 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+tuning: {}
+  # This section contains Redpanda tuning parameters.
+  # Each parameter below is set to their default values.
+  # Remove the curly brackets above if you uncomment any parameters below.
+  #
+  # Setup NIC IRQs affinity, sets up NIC RPS and RFS, sets up NIC XPS, increases socket
+  # listen backlog, increases the number of remembered connection requests, bans the
+  # IRQ Balance service from moving distributed IRQs.
+  # tune_network: false
+  #
+  # Sets the preferred I/O scheduler for given block devices.
+  # It can work using both the device name or a directory, in which the device where
+  # directory is stored will be optimized. Sets either 'none' or 'noop' scheduler
+  # if supported.
+  # tune_disk_scheduler: false
+  #
+  # Disables IOPS merging.
+  # tune_disk_nomerges: false
+  #
+  # tune_disk_irq must be false in container environments and will be ignored.
+  #
+  # Installs a systemd service to run fstrim weekly, or starts the default fstrim service
+  # which comes with most Linux distributions.
+  # tune_fstrim: false
+  #
+  # tune_cpu must be false in container environments and will be ignored.
```

Comment:
This section was expanded to include all `tune_` parameters from the documentation.

Two parameters are mentioned and ignored: `tune_disk_irq` and `tune_cpu`. I've seen messages saying these parameters cannot be used in a container. Let me know if there are other tune parameters that should also be ignored.

Reply:
I think only aio_events, ballast_file and clocksource work in a containerized environment. Just ran a test and got:

```
$ redpanda git:(v2) kubectl exec -it -n redpanda redpanda-0 -- rpk redpanda tune all                         
Defaulted container "redpanda" out of: redpanda, set-datadir-ownership (init), redpanda-configurator (init)
got an error while writing 1 to /proc/sys/vm/swappiness: open /proc/sys/vm/swappiness: read-only file system
  TUNER                  APPLIED  ENABLED  SUPPORTED  ERROR                                                      
  aio_events             true     true     true                                                                  
  ballast_file           true     true     true                                                                  
  clocksource            true     true     true                                                                  
  coredump               false    true     true       open                                                       
                                                      /proc/sys/kernel/core_pattern:                             
                                                      read-only file system                                      
  cpu                    false    true     true       open                                                       
                                                      /sys/devices/system/cpu/cpufreq/policy0/scaling_governor:  
                                                      read-only file system                                      
  disk_irq               false    true     true       no such file or                                            
                                                      directory                                                  
  disk_nomerges          false    true     false      no such file or                                            
                                                      directory                                                  
  disk_scheduler         false    true     false      no such file or                                            
                                                      directory                                                  
  disk_write_cache       false    true     false      no such file or                                            
                                                      directory                                                  
  fstrim                 false    true     false      dial unix                                                  
                                                      /var/run/dbus/system_bus_socket:                           
                                                      connect: no such file or                                   
                                                      directory                                                  
  net                    false    true     true       open                                                       
                                                      /proc/sys/net/core/rps_sock_flow_entries:                  
                                                      no such file or directory                                  
  swappiness             false    true     true       open                                                       
                                                      /proc/sys/vm/swappiness:                                   
                                                      read-only file system                                      
  transparent_hugepages  false    true     true       open                                                       
                                                      /sys/kernel/mm/transparent_hugepage/enabled:               
                                                      read-only file system    
```
I used V2 branch to setup a 3 node cluster using kind

Reply:
Ahh that's a great that you can find out what can be ran in a container environment based on running the command. I'll handle all these invalid parameters the same as `tune_disk_irq` and `tune_cpu`.